docs/README.md

```diff
@@ -1,38 +1,28 @@
-# Product Availability
-
-## Description
-
-The Product Availability component shows different messages set by the user, depending on the available quantity of the product.
-
-It is only available to be used on the `store.product` block, in your product page.
+📢 Use this project, [contribute](https://github.com/vtex-apps/product-availability) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request
+# Product Availability
 
-## Table of Contents
+<!-- DOCS-IGNORE:start -->
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+<!-- DOCS-IGNORE:end -->
 
-- [Usage](#usage)
-  - [Blocks API](#blocks-api)
-    - [Configuration](#configuration)
-  - [Styles API](#styles-api)
-    - [CSS Namespaces](#css-namespaces)
-- [Troubleshooting](#troubleshooting)
-- [Tests](#tests)
+The Product Availability app displays text messages regarding the in-stock quantity for products. 
```

Comment:
My suggestion is to add something about "Flash sales" for SEO reasons. Some US clients refer to this feature as a "flash sales" component

Reply:
Hmmm, I don't think I fully understood the message, @denisebiscaro! What do you mean?